### DISCUSSION TITLE
External Auths

### DISCUSSION THREAD

**Discussion by Fynnstw** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#236] [Q&A] [answered]
How do you this with the Account Id,psn name, xbox name,switch name and steam name

**Comment by LeleDerGrasshalmi** [comment]
excuse me, what?

**Reply by Fynnstw** [reply]
How to do this with the account id xbox name psn name steam name and switch mane because not every account has the same id and the same name etc. I would like to do a command like bwi wafer bot's account find command and Every other epic acc has a different name and a different ID. Maybe you now know what I mean. If not, I can send you a photo

**Reply by Fynnstw** [reply]
@LeleDerGrasshalmi

**Reply by LeleDerGrasshalmi** [reply]
every account has a different id, i dont really understand what you mean.
But various account lookup apis are documented in this repo

**Reply by qwlwisch** [reply]
![Screenshot_2024-01-23-21-11-35-39_3aea4af51f236e4932235fdada7d1643](https://github.com/LeleDerGrasshalmi/FortniteEndpointsDocumentation/assets/156588628/fed7e988-f3dc-44f9-a0c6-d8f2c7cc2dcb)
![Screenshot_2024-01-23-15-59-32-89_50ef9f5a0f3fc24b6f0ffc8843167fe4](https://github.com/LeleDerGrasshalmi/FortniteEndpointsDocumentation/assets/156588628/fcd7e580-39fa-4dd9-9517-9cefb524a4d4)
I think he means that

**Reply by LeleDerGrasshalmi** [reply]
as i said, various lookup apis can be found in this repo.

**Reply by Fynnstw** [reply]
I find it under the French apartment

**Reply by LeleDerGrasshalmi** [reply]
???

**Reply by Fynnstw** [reply]
where can I find it

**Reply by LeleDerGrasshalmi** [reply, marked as answer]
You havent said what you want. Someone else only said what you may have meant,
